charts/istio-policy/templates/grafana-envoy-filter.yaml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2025 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+{{- if .Values.enableGrafanaHeaderSizeLimits }}
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: grafana-header-size-limits
+  namespace: istio-system
+spec:
+  workloadSelector:
+    labels:
+      app.kubernetes.io/name: grafana
+  configPatches:
+  - applyTo: NETWORK_FILTER
+    match:
+      context: SIDECAR_INBOUND
+      listener:
+        filterChain:
+          filter:
+            name: "envoy.filters.network.http_connection_manager"
+    patch:
+      operation: MERGE
+      value:
+        typed_config:
+          "@type": "type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager"
+          max_request_headers_kb: 128
```

Comment:
Do we want to make this a configurable size?

Reply:
Absolutely. Will do that change

Reply:
Updated